tests/MonitoringFunctions.Test/TestDirectJsonMappingResolver.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Kusto.Data.Common;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MonitoringFunctions.DataService.Kusto;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MonitoringFunctions.Test
+{
+    /// <summary>
+    /// Contains tests that make sure <see cref="DirectJsonMappingResolver"/> can correctly create
+    /// <see cref="ColumnMapping"/>s for models.
+    /// </summary>
+    [TestClass]
+    public class TestDirectJsonMappingResolver
+    {
+        private class BasicMappingModel : IKustoTableRow
+        {
+            public float someFloat { get; set; }
+
+            public string someString { get; set; }
+        }
+
+        private class CommonMappingModel : IKustoTableRow
```

Comment:
this could be a child of ```ComplexMappingModel```

Reply:
Made `ComplexMappingModel` inherit from `CommonMappingModel` and modified the members a bit. This will better test the abilities of our column mapping resolver.